layouts/partials/css/custom.css

```diff
@@ -186,13 +198,15 @@ code, kbd, pre, samp {
   padding: 0.5em;
 }
 
-.post a {
-  color: #c05b4d;
+.post a,
+.post-content a {
+  color: var(--link-color);
   text-decoration: none;
 }
 
-.post a:hover {
-  color: #a5473a;
+.post a:hover,
+.post-content a:hover {
```

Comment:
Same as above, if you could remove `.post-content a:hover` addition that would be great!